tests/dummy/app/models/global-navigation.json

```diff
@@ -266,7 +266,7 @@
       }
     ],
     "user": {
-      "avatar_url": "/design-system/images/FANDOM-Avatar.jpg",
+      "avatar_url": "/images/FANDOM-Avatar.jpg",
```

Comment:
this url does not work when you deploy to devbox (and probably on fandomdesignsystem.com too)